src/interpolators.jl

```diff
@@ -9,28 +7,41 @@ Interface for all types <: Interpolator:
 abstract type Interpolator end
 
 """
-    ClaughTochter(fill_value=NaN, tol=1e-6, maxiter=400, rescale=false)
+    CloughTocher(fill_value=NaN, tol=1e-6, maxiter=400, rescale=false)
 
 Piecewise cubic, C1 smooth, curvature-minimizing interpolant in 2D.
-Find more detailed docs in [SciPy.interpolate.CloughTocher2DInterpolator](https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.CloughTocher2DInterpolator.html).
-Slow, but yields the smoothest interpolation.
+Find more detailed docs in CloughTocher2DInterpolator.jl, which is a reimplementation of  [SciPy.interpolate.CloughTocher2DInterpolator](https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.CloughTocher2DInterpolator.html).
```

Comment:
```suggestion
Find more detailed docs in CloughTocher2DInterpolator.jl.
```

I don't think we need to mention where this comes from -- let that package's docs say that.